src/test/functional/features/sole-conditional-order.feature

```diff
@@ -52,21 +52,6 @@ Feature: Sole conditional order
     And I click "Continue"
     Then the page URL should be "/hub-page"
 
-    Given I set the case state to "AwaitingClarification"
-    When I click "Sign out"
-    And I login with applicant "1"
-    When I go to "/provide-information-to-the-court"
-    Then the page should include "Upload any documents"
-
-    When I clear the form
-    Given I select "If the court wants you to explain something or provide additional information then write your response here. If the court has just asked you to upload documents then you do not have to write anything, unless you think it’s useful information."
-    And I type "test details"
-    And I select "I cannot upload some or all of my documents"
-    Then the page should include "You can post your documents to the court"
-
-    When I click "Continue"
-    Then the page URL should be "/hub-page"
```

Comment:
how comes you deleted this?

Reply:
this test is repeated on the sole-hub-page.feature

Reply:
if you think its worth having it here and deleting it in the hub page one then i can go for that but i dont think its worth having both.